src/RWXLoader.js

```diff
@@ -425,6 +425,53 @@ function applyTextureToMat( threeMat, folder, textureName, textureExtension = '.
 
 	}
 
+	// Load normal map
+
+	if ( normalName != null ) {
+
+	  const urlRes = urlRegex.exec( normalName );
+	  const normalPath = urlRes ? normalName : folder + '/' + normalName + textureExtension;
+
+	  loadingPromises.push( new Promise( ( resolve, reject ) => {
+
+	    loader.load( normalPath, ( normalMap ) => {
+
+	      normalMap.wrapS = textureWrapping;
+	      normalMap.wrapT = textureWrapping;
+	      threeMat.normalMap = normalMap;
+	      threeMat.needsUpdate = true;
+
+	      resolve( normalMap );
+
+	    }, null, reject );
+
+		} ) );
+
+	}
```

Comment:
Mind the indentation here: it should be tabs instead of spaces, just like the specular map handling below  (you can also remove the extra line break after the comment)